prow/github/client.go

```diff
@@ -636,6 +642,8 @@ type ClientOptions struct {
 	AppID         string
 	AppPrivateKey func() *rsa.PrivateKey
 
+	IsGitHubAction bool
```

Comment:
is there a way to automatically determine this binary runs in GH actions?

Reply:
It is up to the code that is instantiating the github client to indicate if it is being run from a Github Action.
The `IsGitHubAction` field will default to false if not explicitly set.

Reply:
@chaodaiG thank you for asking the question 🙏🏻 btw. I appreciate you for asking it! 🙂